docker/swift-t/Dockerfile

```
@@ -4,37 +4,20 @@
 # http://swift-lang.github.io/swift-t/guide.html#install_source
 
 # The ExaWorks SDK Base
-FROM 136a88385611
+FROM exaworks/sdk-base
 
 # Customize the Swift/T root directory here:
 ENV SWIFT_ROOT=/opt/swift-t
 
-# Install dependencies
-RUN yum -y install mpich-devel # OpenMPI is fine too
-RUN yum -y install swig
-RUN yum -y install zsh
-RUN wget https://prdownloads.sourceforge.net/tcl/tcl8.6.11-src.tar.gz
-RUN tar xfz tcl8.6.11-src.tar.gz
-RUN cd tcl8.6.11/unix \
-  && ./configure --prefix=/opt/tcl-8.6.11 \
-  && make -j binaries libraries \
-  && make install-binaries install-libraries install-headers
-
-# Configure environment
-ENV PATH=/usr/lib64/mpich/bin:/opt/tcl-8.6.11/bin:$PATH
-
-# Get Swift/T and setup
+# Setup build environment
 RUN mkdir -pv /tmp/build-swift-t
-WORKDIR /tmp/build-swift-t
-RUN git clone https://github.com/swift-lang/swift-t.git
-RUN cd swift-t/dev/build \
-    && cp -v swift-t-settings.sh.template swift-t-settings.sh \
-    && sed -i "/SWIFT_T_PREFIX=/s@=.*@=$SWIFT_ROOT@" swift-t-settings.sh \
-    && sed -i "/PARALLELISM=/s@=.*@=4@"              swift-t-settings.sh
+WORKDIR       /tmp/build-swift-t
+COPY install-deps.sh    .
+COPY install-swift-t.sh .
+COPY test-sanity.sh     .
```

These `COPY`s can also be merging into a single one:

```suggestion
COPY install-deps.sh install-swift-t.sh test-sanity.sh .
```

Reference: https://docs.docker.com/engine/reference/builder/#copy

---

I accidentally resolved this- I can make this change.